whitepaper.md

```diff
@@ -327,67 +347,70 @@ curl -g 'http://localhost:9090/api/v1/query_exemplars?query=test_exemplar_metric
+Generally speaking, "closed" box monitoring refers to observing a system from the outside where the operator has no control or knowledge of the inner workings of the system. "open" box monitoring, on the other hand, refers to the more "traditional" concept of monitoring an application or system that you control and understanding how it functions, so you're able to make better decisions on how it should be observed through the Three Pillars of Observability.
 
 ### Implementing SLIs, SLOs and SLAs
 
-Implementing SLI, SLO and SLA metrics lets you measure service quality and customer happiness objectively. More so over, it provides a common set of terminologies between different functions like business, product and engineering within an org. Engineering time is a scarce resource within any organization, but everyone feels like their problem is a burning problem. SLOs make such conversations more data-driven because everyone understands the business consequences of breaching SLOs. While solving internal conflicts, it also makes you more customer-obsessed by providing meaningful abstractions that enable meaningful and actionable alerting.
+Implementing SLI, SLO and SLA metrics lets you measure service quality and customer happiness objectively. More so over, it provides a common set of terminologies between different functions like business, product and engineering within an org. Engineering time is scarce in any organization, but everyone feels like their problem is a burning problem. SLOs make such conversations more data-driven because everyone understands the business consequences of breaching SLOs. While solving internal conflicts, it also makes you more customer-obsessed by providing meaningful abstractions that enable meaningful and actionable alerting.
 
 Before we deep dive into the implementation details, we should get the definitions clear as they can be fairly confusing and sometimes be used interchangeably.
 
 * Service Level Indicator (SLI): An SLI is a service level indicator—a carefully defined quantitative measure of some aspect of the level of service that is provided.
-* Service Level Objective (SLO): An SLO is a service level objective: objective for how often you can afford for it to fail. a target value or range of values for a service level that is measured by an SLI
+* Service Level Objective (SLO): An SLO is a service level objective: objective for how often you can afford for it to fail. It is a target value or range of values for a service level that an SLI measures.
 * Service Level Agreement (SLA): a business contract that includes consequences of violating the SLO. This is a targeted percentage
-Error budget: tolerance for failed events over a period of time determined by SLO. This is 100% minus the SLO
+  Error budget: tolerance for failed events over a period of time determined by SLO. This is 100% minus the SLO.
```

Comment:
Should `Error budget` be a definition on same level as SLI/SLO/SLA? i.e. is it missing the `*`?